webpack/components/AnsibleHostDetail/components/AnsibleVariableOverrides/AnsibleVariableOverridesTableHelper.js

```diff
@@ -26,3 +29,50 @@ export const formatValue = variable => {
       return value;
   }
 };
+
+const joinErrors = errors => errors.map(err => err.message).join(', ');
+
+const onCompleted = (toggleModal, showToast) => data => {
+  toggleModal();
+  const { errors } = data.deleteAnsibleVariableOverride;
+  if (Array.isArray(errors) && errors.lenght > 0) {
```

Comment:
What does `errors` contain if there is no errors? what about when there is a network error? Also, there is a typo here
```suggestion
  if (Array.isArray(errors) && errors.length > 0) {
```

Reply:
`errors` are empty array if there are no errors (and errors field is included in response).

Unexpected failures should be handled by `onError`.